### DISCUSSION TITLE
Not an Issue, looking for a place to ask a question about running within docker

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [bobokun/actualtap-py#96] [General]
Hi I stumbled upon this from the Actual Budget documentation pages.  I was going to give it a try via docker and looked at the docker compose file and it looks like all I need to do is change the directory for the config.  

I'm still very new to docker but I'm wondering if that's really all that is needed or if I need to also change the config file - presumably after it's installed via the docker compose?  I was also planning on installing it via portainer - assume I can take the docker compose contents and paste it into a new stack?

**Comment by Lunchtime0614** [comment]
I figured out how to get it running....but now I'm wondering about the tap to pay card mappings.  I see in the config.yml that there's a name for savings, checking, credit_card....are those the names of the cards in the iOS wallet - my card names are very long and have spaces, if I were to map those how would I do it...say for example iOS shows the name as "Capital Bank Signature Checking"

**Comment by marleyjaffe** [comment]
I believe this comment answers your question. https://github.com/bobokun/actualtap-py/issues/35#issuecomment-2415300527
